apps/meteor/server/lib/http/call.ts

```diff
@@ -177,12 +184,21 @@ function _call(httpMethod: string, url: string, options: HttpCallOptions, callba
 function httpCallAsync(httpMethod: string, url: string, options: HttpCallOptions, callback: callbackFn): void;
 function httpCallAsync(httpMethod: string, url: string, callback: callbackFn): void;
 function httpCallAsync(httpMethod: string, url: string, optionsOrCallback: HttpCallOptions | callbackFn = {}, callback?: callbackFn): void {
-	// If the options argument was ommited, adjust the arguments:
+	// If the options argument was omitted, adjust the arguments:
 	if (!callback && typeof optionsOrCallback === 'function') {
 		return _call(httpMethod, url, {}, optionsOrCallback as callbackFn);
 	}
 
 	return _call(httpMethod, url, optionsOrCallback as HttpCallOptions, callback as callbackFn);
 }
 
-export const httpCall = Meteor.wrapAsync(httpCallAsync);
+export const httpCall = async (httpMethod: string, url: string, options: HttpCallOptions) => {
```

Comment:
I wonder if we should let this as is but start migrating code using this function to use `fetch` instead.

Reply:
I did the migration to fetch already where it was possible. Had to change this to use Promise.await for the integrations part and let us decide the solution for the integrations later

Reply:
Thinking better now, we may need to wait Kevin's work replacing the HTTPs by fetch, I didn't realized the other methods as `HTTP.post` were calling the `HTTP.call` as well :(